target/linux/realtek/files-5.10/drivers/net/ethernet/rtl838x_eth.c

```diff
@@ -2411,18 +2412,30 @@ static int __init rtl838x_eth_probe(struct platform_device *pdev)
 		goto err_free;
 	}
 
-	priv->membase = dmam_alloc_coherent(&pdev->dev, rxrings * rxringlen * RING_BUFFER
-				+ sizeof(struct ring_b), &priv->membase_dma, GFP_KERNEL);
+	priv->rxspace = dmam_alloc_coherent(&pdev->dev, rxrings * rxringlen * RING_BUFFER,
+					    &priv->rxspace_dma, GFP_KERNEL);
+	if (!priv->notify) {
```

Comment:
leaving a note for myself mostly, but this surey must have been:
```suggestion
	if (!priv->rxspace) {
```